plugins/CoreAdminHome/tests/Integration/ArchiveReportsMetricsTimerTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\CoreAdminHome\tests\Integration;
+
+use Piwik\Common;
+use Piwik\Db;
+use Piwik\Plugins\ArchivingMetrics\Timer;
+use Piwik\Plugins\CoreAdminHome\API as CoreAdminHomeAPI;
+use Piwik\Tests\Framework\Fixture;
+use Piwik\Tests\Framework\TestCase\IntegrationTestCase;
+
+/**
+ * @group CoreAdminHome
+ * @group CoreAdminHome_ArchiveReportsMetricsTimer
+ * @group Plugins
+ */
+class ArchiveReportsMetricsTimerTest extends IntegrationTestCase
+{
+    public static function setUpBeforeClass(): void
+    {
+        if (self::$fixture) {
+            self::$fixture->extraPluginsToLoad[] = 'ArchivingMetrics';
+        }
```

Comment:
```suggestion
        self::$fixture->extraPluginsToLoad[] = 'ArchivingMetrics';
```

To get a really early error if that fixture is ever missing. And not later when the database entry assertions fail.